### ISSUE TITLE
Create (better) documentation on how to create themes / Create repo for themes

### ISSUE THREAD

**Issue by nikisix** [Cockatrice/Cockatrice#1973]
Which sourcefile(s) should I edit to make the background dark and the font light colors?

**Comment by leematos** [comment]
I have a dark theme that I can upload. This guide covers where to find the theme stuff: https://github.com/Cockatrice/Cockatrice/wiki/Themes

**Comment by ctrlaltca** [comment]
Maybe we should start considering the idea of a theme repository or at least refresh Trice's bundled themes with new additions?

**Comment by Daenyth** [comment]
I have considered a theme repo, and I'm in favor :D

I've been toying with elm lately, this might be a good project for me to do. Seems like it can be small scope and focused. I wonder if I could run it off heroku....

Let's start speccing out an MVP for it.

**Comment by Jinxstands4Jinx** [comment]
Can we rewrite that guide..maybe in english.

**Comment by Daenyth** [comment]
@Jinxstands4Jinx It's obviously already in English. If you're trying to be constructive rather than annoying, please give specific feedback. Perhaps name a specific section that's not clear, or a question you have that you weren't able to find the answer for (in a new ticket please so we can keep this one focused). I know it's not ideal, but it is also a wiki, so feel free to improve the wording or layout.

**Comment by Jinxstands4Jinx** [comment]
Sorry but I tried to read that wiki and it is not in English. Maybe what I
mean to say is could you please dumb it down some more not all cockatrice
users have  a masters in computer programming. I know how to set the Theme
that i have only by doing trail and error over several hours of work. I was
not trying to be annoying but i was asking that it be dumbed down some.

**Comment by Nightfirecat** [comment]
Can you cite specific examples? There are certain details about creating themes which will be complicated (in its current form, it can't be made more simple), but most of it is simple enough&mdash;read the details/instructions at the head of each table then use the tables to understand which images correspond to each section.

**Comment by Daenyth** [comment]
I'm taking from Jinx's post that we should split the page into sections about using and creating themes.

**Comment by Jinxstands4Jinx** [comment]
I just know that I found exactly No help in that wiki. i read it 4 times
and it made Zero sense. I resorted to trial and error methods to figure out
how to use the themes. and once I got close to the theme I wanted I stopped
and just "walked away" Maybe put a youtube video along with the wiki?

**Comment by ctrlaltca** [comment]
> please give specific feedback. Perhaps name a specific section that's not clear, or a question you have that you weren't able to find the answer for

> Can you cite specific examples

@Jinxstands4Jinx  Feel free continue to say that the wiki page sucks, but we already know it :)
Can you please elaborate instead on what you would like to see in that page, what informations you are missing, etc?

**Comment by Jinxstands4Jinx** [comment]
How about a link to a You-Tube video showing people how to set up a Default theme? That way they can just follow along.

**Comment by Daenyth** [comment]
Not a bad idea

**Comment by Daenyth** [comment]
Is there any action for us to take on this? If not we can close this ticket.

**Comment by Jinxstands4Jinx** [comment]
is there an Intro video made to set up a default theme?

**Comment by ZeldaZach** [comment]
Dupe #4087
